dlpy/speech.py

```diff
@@ -0,0 +1,258 @@
+from dlpy.speech_utils import *
```

Comment:
In all method docstrings in this file, make sure you put a space before and after the ':' of every parameter description.  For example, 'data_path: string' should be 'data_path : string'.  If I recall correctly, the Sphinx doc renderer is picky about that.